<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_PlaylistPreferences</class>
 <widget class="QWidget" name="GUI_PlaylistPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>373</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cb_show_numbers">
     <property name="text">
      <string>Show numbers</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="le_expression"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lab_title">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Look</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>'italic text'</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Example</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string notr="true">%nr%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string notr="true">%artist%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>*bold text*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>track number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string notr="true">%album%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string notr="true">%title%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>artist</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>album</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string notr="true"> %nr% - '%title%' - %artist%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="4">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_default">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
